### PR TITLE
Added logger

### PR DESCRIPTION
**Pull request by Daniel1004K**

### CODE REVIEW

File: src/main/java/mate/academy/service/AuthenticationServiceImpl.java [hunk 1]
```diff
@@ -2,22 +2,28 @@
...
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final Logger logger = LogManager.getLogger(AuthenticationServiceImpl.class);
+
     @Override
     public User login(String login, String password) throws AuthenticationException {
-        //TODO: add corresponding log message about method login was called
+        logger.debug("Method 'login' was called. Params: login = {}",login);
```
Comment (Veronika-Alekseienko) [inline src/main/java/mate/academy/service/AuthenticationServiceImpl.java:13]:
```suggestion
        logger.debug("Method 'login' was called. Params: login = {}", login);
```

File: src/main/java/mate/academy/service/OrderServiceImpl.java [hunk 2]
```diff
@@ -4,27 +4,28 @@
...
 
     private List<Product> getAllProductsFromShoppingCart(Long userId) {
-        // NOTE: In production ready code this method should fetch data from DB
-        // For test purpose we simplify this method and return dummy data
         Product iphone = new Product("iPhone X", BigDecimal.valueOf(1199));
         Product macBook = new Product("MacBook Air 2020", BigDecimal.valueOf(1399));
         Product xiaomi = new Product("Xiaomi 12", BigDecimal.valueOf(499));
         List<Product> products = List.of(iphone, macBook, xiaomi);
-        // TODO: add log message about successful fetched data from DB
+        logger.info("Data was fetched successful from DB. Params : userId = {}",userId);
```
Comment (Veronika-Alekseienko) [inline src/main/java/mate/academy/service/OrderServiceImpl.java:28]:
```suggestion
        logger.info("Data was fetched successful from DB. Params : userId = {}", userId);
```

File: src/main/java/mate/academy/service/OrderServiceImpl.java [hunk 3]
```diff
@@ -4,27 +4,28 @@
...
     @Override
     public Order completeOrder(Long userId) {
-        // TODO: add log message about method completeOrder was called
+        logger.debug("Method completeOrder was called. Params: userId = {}",userId);
         List<Product> products = getAllProductsFromShoppingCart(userId);
         Order order = new Order(products, userId);
-        // NOTE: In production ready code this order identifier should be generated by DB
-        // For test purpose we simplify this and return dummy data
         order.setOrderId(1L);
+        logger.info("Method 'completeOrder' successfully completed. Params = {}",userId);
```
Comment (Veronika-Alekseienko) [inline src/main/java/mate/academy/service/OrderServiceImpl.java:19]: same redaundent line

File: src/main/java/mate/academy/service/OrderServiceImpl.java [hunk 4]
```diff
@@ -4,27 +4,28 @@
...
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
+    private static final Logger logger = LogManager.getLogger(OrderServiceImpl.class);
+
     @Override
     public Order completeOrder(Long userId) {
-        // TODO: add log message about method completeOrder was called
+        logger.debug("Method completeOrder was called. Params: userId = {}",userId);
```
Comment (Veronika-Alekseienko) [inline src/main/java/mate/academy/service/OrderServiceImpl.java:15]:
```suggestion
        logger.debug("Method completeOrder was called. Params: userId = {}", userId);
```

File: src/main/java/mate/academy/service/AuthenticationServiceImpl.java [hunk 5]
```diff
@@ -2,22 +2,28 @@
...
         User user = findByLogin(login);
         if (!user.getPassword().equals(password)) {
             throw new AuthenticationException("Username or password are incorrect");
         }
+        logger.info("Method 'login' successfully completed. Params: login = {}",login);
         return user;
     }
 
     private User findByLogin(String login) {
+        logger.debug("Method 'findByLogin' was started ");
```
Comment (Veronika-Alekseienko) [inline src/main/java/mate/academy/service/AuthenticationServiceImpl.java:23]: u shouldn't do it here

File: src/main/java/mate/academy/service/AuthenticationServiceImpl.java [hunk 6]
```diff
@@ -2,22 +2,28 @@
...
+
     @Override
     public User login(String login, String password) throws AuthenticationException {
-        //TODO: add corresponding log message about method login was called
+        logger.debug("Method 'login' was called. Params: login = {}",login);
         User user = findByLogin(login);
         if (!user.getPassword().equals(password)) {
             throw new AuthenticationException("Username or password are incorrect");
         }
+        logger.info("Method 'login' successfully completed. Params: login = {}",login);
```
Comment (Veronika-Alekseienko) [inline src/main/java/mate/academy/service/AuthenticationServiceImpl.java:18]: i think it's redaundent line

File: src/main/java/mate/academy/Main.java [hunk 7]
```diff
@@ -6,17 +6,22 @@
...
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
         User user;
         try {
             user = authenticationService.login("bob", "1234");
         } catch (AuthenticationException e) {
-            e.printStackTrace();
+            logger.error("Can't authenticate", e);
             return;
         }
+        logger.info("Authentication was completed successfully.");
```
Comment (Veronika-Alekseienko) [inline src/main/java/mate/academy/Main.java:24]: same redaundent line

**Review by Veronika-Alekseienko — CHANGES_REQUESTED** [review]
Good job!)

**Review by Veronika-Alekseienko — APPROVED** [review]

File: src/main/java/mate/academy/service/AuthenticationServiceImpl.java [hunk 8]
```diff
@@ -2,21 +2,26 @@
...
+
     @Override
     public User login(String login, String password) throws AuthenticationException {
-        //TODO: add corresponding log message about method login was called
+        logger.debug("Method 'login' was called. Params: login = {}", login);
         User user = findByLogin(login);
         if (!user.getPassword().equals(password)) {
             throw new AuthenticationException("Username or password are incorrect");
         }
+        logger.info("Method 'login' successfully completed. Params: login = {}", login);
```
Comment (KseniiaMakarova) [inline src/main/java/mate/academy/service/AuthenticationServiceImpl.java:18]: тут также можно юзать debug

File: src/main/java/mate/academy/service/OrderServiceImpl.java [hunk 9]
```diff
@@ -4,27 +4,28 @@
...
     @Override
     public Order completeOrder(Long userId) {
-        // TODO: add log message about method completeOrder was called
+        logger.debug("Method completeOrder was called. Params: userId = {}", userId);
         List<Product> products = getAllProductsFromShoppingCart(userId);
         Order order = new Order(products, userId);
-        // NOTE: In production ready code this order identifier should be generated by DB
-        // For test purpose we simplify this and return dummy data
         order.setOrderId(1L);
+        logger.info("Method 'completeOrder' successfully completed. Params = {}", userId);
```
Comment (KseniiaMakarova) [inline src/main/java/mate/academy/service/OrderServiceImpl.java:19]: и тут

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request closed without merging**